### PR TITLE
change boundary conditions to dirichlet 0 for nsinker benchmark

### PR DESCRIPTION
**Pull request by quangx**
nsinker.cc says the benchmark is meant to be as defined in Rudi et al. (2017), which uses Dirichlet 0 boundary conditions. This is also consistent with the boundary conditions implemented for the sinker problem in Clevenger, Heister (2021).

### CODE REVIEW

**Comment by tjhei** [comment]
You will need to update test output.

/rebuild

File: benchmarks/nsinker/nsinker.prm [hunk 1]
```diff
@@ -39,7 +39,7 @@ subsection Geometry model
 end
 
 subsection Boundary velocity model
-  set Tangential velocity boundary indicators = left,right,bottom,front,back
+  set Zero velocity boundary indicators = left,right,bottom,front,back,top
```
Comment (bangerth) [inline benchmarks/nsinker/nsinker.prm:42]: Let's put like to like:
```suggestion
  set Zero velocity boundary indicators = left,right,bottom,top,front,back
```

**Comment by bangerth** [comment]
I admit that my request is pedantic, but I'd still like to ask for one change.

**Comment by quangx** [comment]
I did the reorder, ready to be reviewed again.

**Review by gassmoeller — APPROVED** [review]

**Pull request merged**